HW2/src/test/java/com/example/hw2/controller/IntegrationTest.java

```diff
@@ -0,0 +1,139 @@
+package com.example.hw2.controller;
+
+import com.example.hw2.entity.Author;
+import com.example.hw2.servise.AuthorService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureWebMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.context.WebApplicationContext;
+
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+@SpringBootTest
+@AutoConfigureWebMvc
+public class IntegrationTest {
+
+    @Autowired
+    private WebApplicationContext webApplicationContext;
+    @Autowired
+    private AuthorService authorService;
+    private MockMvc mockMvc;
+
+    @BeforeEach
+    public void setUp() {
+        mockMvc = MockMvcBuilders.webAppContextSetup(webApplicationContext).build();
+    }
+
+    @Test
+    public void getById() throws Exception {
```

Comment:
The test class name is not providing any information about API that you're testing.

So, please specify this information via test method and via `@DisplayName` and provide information about test case.

`Author_getById_shouldReturn...`

Reply:
Thank you, done